src/League/OAuth2/Server/Grant/GrantTypeInterface.php

```diff
@@ -28,6 +24,11 @@
     public function __construct(Authorization $authServer = null);
 
     /**
+     * @param TokenGeneratorInterface $generator
+     */
+    public function setTokenGenerator(TokenGeneratorInterface $generator);
```

Comment:
Using setter injection over constructor injection here as it's easier for an integrator to set a custom token generator.